README.md

```diff
@@ -1,20 +1,19 @@
 # @comet/admin
 
-## Developoment
+## Development
 
 ### Requirements
 
--   [docker & docker-compose](https://docs.docker.com/compose/)
+-   [nvm](https://github.com/nvm-sh/nvm)
 
-### Run build and storybook
-
-    docker-compose up # on linux
-    docker-compose -f docker-compose.yml -f docker-compose.mac.yml up # on OSX
+Run `install.sh` once.
 
-### use yarn
+### Run build and storybook
 
-    docker-compose exec comet-admin bash -c "yarn --help"
-    docker-compose exec comet-admin bash -c "cd packages/admin && yarn --help"
+```
+npm run start
```

Comment:
Shouldn't this be `yarn start`? Or does `lerna run ...` use yarn under the hood anyway? As far as I know, we need to use yarn in combination with lerna.

Reply:
btw: do we still need yarn - as a recent npm relase added support for workspaces (afaik compatible with yarn)

the whole nvm setup does somehow ignore yarn anyway :)

Reply:
Found out that lerna uses yarn as configured in `lerna.json`, so please ignore my remark. 😅

Reply:
should indeed be `yarn ...`, fixed now